airflow/task_runner/base_task_runner.py

```diff
@@ -97,7 +97,11 @@ def _read_task_logs(self, stream):
                 line = line.decode('utf-8')
             if len(line) == 0:
                 break
-            self.log.info(u'Subtask: %s', line.rstrip('\n'))
+            self.log.info(
+                u'Subtask %d: %s',
+                self._task_instance.job_id,
```

Comment:
@wrp just realized that we typically identify a task by its task_id, so may be better to use `self._task_instance.task_id` here instead. The job_id is an auto-increment primary key of a SchedulerJob/BackfillJob/LocalTaskJob, which for a given dagrun is the same for all tasks.

Reply:
Good point.  I think it would be helpful to have both in the log header.